lua/packer/plugin_utils.lua

```diff
@@ -208,9 +208,21 @@ plugin_utils.load_plugin = function(plugin)
   end
 end
 
+plugin_utils.is_remote_plugin = function(plugin)
```

Comment:
Do we use this anywhere?

Reply:
@wbthomason nope it was from the previous commit, I left it in place incase it might be useful in the future/I wasn't really sure what direction to take this PR in since neither approach resolve the issue 100% for me. I think I'm still trying to get clear on exactly what is happening.

It seems like if I run update whilst the rplugin is still not loaded, then the manifest is created without it's content. So when I next go to load the plugin (having restarted nvim since the original session has sourced the file before it changed) then I see the errors.

Somehow adding it just after packadd doesn't seem to always work though. I need to test this again.